src/components/OutboundLink.js

```diff
@@ -0,0 +1,40 @@
+var React = require('react');
+var assign = require('react/lib/Object.assign');
+
+var NEWTAB = '_blank';
+
+var OutboundLink = React.createClass({
+  displayName: 'OutboundLink',
+  propTypes: {
+    eventLabel: React.PropTypes.string.isRequired
```

Comment:
I decided to add `eventLabel` and make it a required prop since it's a required field for `ga.outboundLink` as well and the label is [the only user input field we are passing to GA](https://github.com/mmmavis/react-ga/blob/fe5c567e790866b08e6b47a5619ee16480365f22/src/index.js#L248-L253).

Also `<OutboundLink eventLabel='foo' ...>` is a lot clear than `<OutboudLink args={label: 'foo'}>` and I think it's less likely that people will forget to pass that prop.